rust/src/push/mod.rs

```diff
@@ -491,6 +495,18 @@ fn test_deserialize_unstable_msc3664_condition() {
     ));
 }
 
+#[test]
+fn test_deserialize_unstable_msc3931_condition() {
+    let json =
+        r#"{"kind":"org.matrix.msc3931.room_version_supports","feature":"org.example.feature"}"#;
+
+    let condition: Condition = serde_json::from_str(json).unwrap();
+    assert!(matches!(
+        condition,
+        Condition::Known(KnownCondition::RoomVersionSupports { feature: _ })
```

Comment:
Any reason to not try to match feature value `org.example.feature` too here ?

Reply:
"Rust yelled at me" 

If there's a way to do that, happy to do so :)

Reply:
So I tried `Cow::Borrowed("org.example.feature")` and the compiler yelling stopped, but the outcome was a failure, so I guess there are Cow subtleties I don't get.
Let's move on it's good enough like that.